ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -26,7 +26,11 @@ import LedgerInstructionField from '../../../components/app/ledger-instruction-f
 import { TokenStandard } from '../../../../shared/constants/transaction';
 import { CHAIN_IDS, TEST_CHAINS } from '../../../../shared/constants/network';
 import ContractDetailsModal from '../../../components/app/modals/contract-details-modal/contract-details-modal';
-import { ICON_NAMES, ButtonIcon } from '../../../components/component-library';
+import {
+  ICON_NAMES,
+  ButtonIcon,
```

Comment:
What are you using `ButtonIcon` for here?

Reply:
No, I haven't added ButtonIcon here instead I imported `Icon` here. In this file, `ButtonIcon` is used for `copy` icon. This PR is to update the tag icon only, hence I didn't update the Copy Icon here.

Reply:
Oh duh it was already imported mb 🤦